docs/api/safety.md

```diff
@@ -62,3 +62,19 @@ By fully disabling safety features, i.e. setting `safety_override = full` the ro
 
 !!! attention 
     The robot may not be able to stop in time if driving at full speed if a cliff is detected!
+
+## Acceleration Limits
+
+The robot exposes its commanding acceleration limits through the `wheel_accel_limit` parameter on the `motion_control` node.
+The velocity commands sent to the wheels will be ramped with the acceleration profile associated with this value.
+The value defaults to its maximum settable `900 mm/s^2`.  If using heavier payloads, it is advisable to decrease acceleration limits.
```

Comment:
@shamlian Is this heavier payloads comment valuable or should I remove it or alternatively, should it be made more specific

Reply:
I think it's valuable. If we do some experimentation (or some math ;-) ) and determine appropriate limits based on loading, we can add specifics here after this PR goes in.

Reply:
We should add a reference to the nominal max payload, (I.e. 9kg right?)

Reply:
I think we want to experiment with numbers above 9kg with decreased accel limits, but aren't ready to make any promises yet, so I am being vague on purpose.

Reply:
I agree that we should leave this language as-is until we run more experiments.